pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -372,3 +380,37 @@ func (o *LoginOptions) serverProvided() bool {
 	_, err := o.ClientConfig.ClientConfig()
 	return err == nil || !clientcmd.IsNoServerFound(err)
 }
+
+func relativizeClientConfigPaths(cfg *clientcmdapi.Config, base string) (err error) {
```

Comment:
There is [`RelativizeMasterConfigPaths`](https://github.com/openshift/origin/blob/master/pkg/cmd/server/api/helpers.go#L15) but it takes a different argument (`MasterConfig`) while we take a `*clientcmdapi.Config` here, don't think we can reuse it. Apart that, we are sharing `MakeAbs` and `MakeRelative` between both functions.